tests/test_subdivisions/test_germany.py

```diff
@@ -5,21 +5,41 @@
     de_address_to_land_code,
     de_hauptstadt_to_land_code,
     de_land_name_to_land_code,
+    de_postcode_to_land_code,
+    de_safe_address_to_land_code,
 )
 
 
 class TestGermany:
     @pytest.mark.parametrize(
         "input_data, expected",
         [
-            ("Katharinenberg 14 – 20 18439 Stralsund ", None),
```

Comment:
Why did you remove that case from the tests? It seems it should now give the correct region via the postcode.